pkg/koordlet/runtimehooks/config.go

```diff
@@ -73,6 +74,11 @@ const (
 	// owner: @saintube @zwzhang0107
 	// alpha: v1.4
 	CoreSched featuregate.Feature = "CoreSched"
+
+	// TerwayQoS enables net QoS feature of koordlet.
+	// owner: @l1b0k
+	// alpha: v1.5
+	TerwayQoS featuregate.Feature = "TerwayQoS"
```

Comment:
should also define NetworkQOS policy here https://github.com/koordinator-sh/koordinator/blob/main/apis/slo/v1alpha1/nodeslo_types.go#L245